### ISSUE TITLE
x/website: Example codes don't have horizontal scroller on mobile devices

### ISSUE THREAD

**Issue by inet56** [golang/go#46519]
### What did you expect to see?
Example codes to have horizontal scroller.

### What did you see instead?
Long lines are wrapped around.

**Comment by seankhliao** [comment]
Can you link to an example?

**Comment by inet56** [comment]
> Can you link to an example?

https://golang.org/pkg/strings/#example_Split

**Comment by dr2chase** [comment]
This needs a better description; do you mean that viewing the website on a mobile device, there is no scroll (and the test does not wrap)?  Can you tell me what sort of mobile device you are using, and if you can tell, the OS/browser version?

Using Chrome on an Android phone with (very) up-to-date software, I see this, which looks to me like the wrapping that you desire:
![Screenshot_20210602-150209](https://user-images.githubusercontent.com/1928999/120537797-b816db00-c3b3-11eb-863f-71382233a41c.png)

**Comment by inet56** [comment]
> This needs a better description; do you mean that viewing the website on a mobile device, there is no scroll (and the test does not wrap)?

Yes. Example codes don't have horizontal scroll and text are (long lines) wrapped.

>Can you tell me what sort of mobile device you are using, and if you can tell, the OS/browser version?

Dev: Symphony i12
OS/Browser: Android 10 Go / Bromite 90

> Using Chrome on an Android phone with (very) up-to-date software, I see this, which looks to me like the wrapping that you desire:

I see the same as you see. But I don't desire line wrapping.

**Comment by inet56** [comment]
I opened a PR - https://github.com/golang/website/pull/56

**Comment by inet56** [comment]
@seankhliao more time to take decision? It's just two lines of change.

**Comment by seankhliao** [comment]
cc @dmitshur
